src/interfaces/IMorpho.sol

```diff
@@ -277,6 +277,11 @@ interface IMorpho {
     /// @notice Executes a flash loan.
     /// @dev Flash loans have access to the whole balance of the contract (the liquidity and deposited collateral of all
     /// markets combined, plus donations).
+    /// @dev Warning: Not ERC-3156 compliant for the following reasons:
+    /// - It would add an unnecessary `flashFee` function since there's no fee on Morpho Blue.
+    /// - `maxFlashLoan` is the token balance of the contract.
+    /// - The receiver of the `assets` is the caller as there's no `receiver` argument in our implementation.
+    /// - The ERC is not widespread.
```

Comment:
hot take we don't care about the reasons 
```suggestion
    /// @dev Warning: Not ERC-3156 compliant.
```

Reply:
I think we do care about the differences if we want to simplify the integrator's work
And it doesn't cost much to include these: the standard is not expected to change nor the flash loan mechanism on Morpho Blue, so no maintenance cost is expected

Reply:
```suggestion
    /// @dev Warning: Not ERC-3156 compliant for the following reasons:
    /// - Morpho Blue doesn't expose the `flashFee` function.
    /// - `maxFlashLoan` is not exposed, but it would be the token balance of the contract.
    /// - The receiver of the `assets` is always the caller.
```